stories/input.js

```diff
@@ -95,18 +96,27 @@ const ExampleInput = () => {
       <Input
         id="id7"
         label="label"
-        onChange={action("onChange")}
+        onChange={action('onChange')}
         type="textarea"
         placeholder="Write here..."
         error="error"
         value="Hello"
         cols={50}
         rows={5}
       />
+      <Title>Number Inputl</Title>
+      <Input
+        id="id8"
+        value={numberValue}
+        onChange={(e) => setNumberValue(e.target.value)}
```

Comment:
This cause a flow error in CI.
Something need to be change in the props definition.

  ```js
  onChange: (any) => void,
  ```
  
> Cannot get `e.target` because property `target` is missing in undefined [1].
>    stories/input.js:111:41
>    111|         onChange={(e) => setNumberValue(e.target.value)}